pkg/operator/mon/mon.go

```diff
@@ -328,6 +329,7 @@ func getNodeInfoFromNode(n v1.Node) (*NodeInfo, error) {
 			break
 		}
 	}
+	nr.Hostname = n.Labels[apis.LabelHostname]
```

Comment:
could you move this line up to just after 324 where the `nr.Name` field is being set?  It's right in the middle of the `nr.Address` setting/checking, which slightly disrupts the readability.